backend/i18n/translator/Squidex.Translator/State/TranslationService.cs

```diff
@@ -124,6 +124,19 @@ public void Save()
 
             Save("__todos.json", translationsTodo);
             Save("__ignore.json", translationToIgnore);
+            SaveKeys();
```

Comment:
I think this line does not make sense. We do not need a an empty key file as a permanent artifact

Reply:
Yea your right, if somebody wants to add a new language they just need to run the command